Antidote/ToxListener.m

```diff
@@ -124,6 +126,24 @@ - (void)controllerDidChangeContent:(RBQFetchedResultsController *)controller
     }
 }
 
+#pragma mark - SubmanagerCalls delegate
+- (void)callSubmanager:(OCTSubmanagerCalls *)callSubmanager receiveCall:(OCTCall *)call audioEnabled:(BOOL)audioEnabled videoEnabled:(BOOL)videoEnabled
+{
+    AppDelegate *delegate = (AppDelegate *)[UIApplication sharedApplication].delegate;
+    UIViewController *visibleVC = [delegate visibleViewController];
+
+    if ([visibleVC isKindOfClass:[CallViewController class]]) {
+        // User is in a middle of call, send some kind of notification?
```

Comment:
Is it possible in current toxav to have incoming call while being in call? Can we have several calls and switch between them?